sqlcipher_flutter_libs/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Tue May 09 11:17:21 CEST 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
```

Comment:
Can we upgrade these to the latest 8.1.1 or is there something else causing an incompatibility?

Reply:
I'm not too experienced with how gradle versions work between the main app and other modules/plugins.
Would using AGP 8 in the plugin break some app using an old version of AGP? Or is Gradle backwards compatible?

I can update to 8.x, it build ok